tests/testthat/test-cast.R

```diff
@@ -116,3 +116,10 @@ test_that("dimensionality matches to" ,{
   expect_dim(vec_cast(x1, x2), c(1, 2))
   expect_dim(vec_cast(TRUE, x2), c(1, 2))
 })
+
+# Factors  ------------------------------------------------------------------
```

Comment:
Can you please put this test in `test-type-factor.R`?

Reply:
I forced pushed for cleaner history